sentry_v2/README.md

```diff
@@ -0,0 +1,33 @@
+# Sentry V2 Sample
+
+This sample shows how to configure [Sentry](https://sentry.io) SDK V2 to intercept and capture errors from the Temporal SDK.
+
+Note: This is a small modification of the original example [sentry (v1)](../sentry) which seems to have
+issues in the Workflow interceptor, as there are modules (`warnings` and `threading`) that Sentry uses that aren't passed through
+to Temporal's sandbox properly, causing the Workflows to fail to execute.
+
+> Sentry V2 only supports Python 3.6 and higher. So if you're on Python 3.5, you may need to refer to the original example
```

Comment:
Temporal Python SDK only supports 3.8 or higher, so this disclaimer is not needed

Reply:
> you cannot have two different poetry groups containing the same package with different versions

Ug, that is unfortunate. I do still think we should have the Poetry groups named after the sample they represent, even if it's a dupe for `sentry_v1` and `sentry_v2` (or there isn't even a `sentry_v1` group at all and pip install is manual). This also may give more credence to the approach of making `sentry_v2` just `sentry`.

Reply:
I've [asked on the Sentry's discussion board](https://github.com/getsentry/sentry-python/discussions/2980#discussioncomment-11291164) what the timeline is for v1 support. Will get back once they provide an answer.

One idea would be to just replace the v1 impl. with the new v2 integration, keep the example called `sentry`, and then add a link in the README to the original v1 commit for posterity?

Reply:
Sure, that would work well. Either way I think we want this to be the primary `sentry` sample since we have to have the dependency group named that.